src/main/java/com/axlabs/neo/grantshares/GrantSharesGov.java

```diff
@@ -499,7 +499,8 @@ public static Object[] execute(int id) {
         if (voteCount * 100 / Storage.getInt(getReadOnlyContext(), MEMBERS_COUNT_KEY) < data.quorum)
             fireErrorAndAbort("Quorum not reached", "execute");
         int yesNoCount = votes.approve + votes.reject;
-        if (votes.approve * 100 / yesNoCount <= data.acceptanceRate) fireErrorAndAbort("Proposal rejected", "execute");
+        if (yesNoCount == 0 || (votes.approve * 100 / yesNoCount <= data.acceptanceRate))
```

Comment:
I was thinking about it during my walk, and what if the `acceptanceRate` is set to 0?

Shouldn't the proposal pass in this case, even if everybody abstained?